### PR TITLE
[MMI] updates the "connect custodian account" flow

### PR DESCRIPTION
**Pull request by zone-live**
Updates the styles of some UI elements of the connect custodian account flow and introduces a new small modal that will point users to their custodian website.

Screenshot 2023-08-03 at 15 56 44
Screenshot 2023-08-03 at 15 56 33

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/helpers/utils/institutional/find-by-custodian-name.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { CUSTODIAN_WEBSITES } from '../../../../shared/constants/institutional/custodian-websites';
+
+export function findCustodianByDisplayName(displayName: string) {
```
Comment (DDDDDanica) [inline ui/helpers/utils/institutional/find-by-custodian-name.ts:3]: would be handy to specify the return type here also

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js [hunk 2]
```diff
@@ -79,81 +79,41 @@ const ConfirmAddCustodianToken = () => {
...
-          </>
-        )}
-
-        <Text padding={4} color={TextColor.textDefault}>
-          {t('token')}
+        <Text
+          padding={4}
+          fontWeight={FontWeight.Bold}
+          variant={TextVariant.headingSm}
+          as="h5"
```
Comment (DDDDDanica) [inline ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js:104]: Optional: you can remove `as` props now, check https://consensys.slack.com/archives/GTQAGKY5V/p1690998425185369

File: ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        custodianName="Qredo"
+        custodianURL="https://qredo.com"
+      />,
+      store,
+    );
+
+    const tokenContainer = screen.getByText(
+      "To connect your accounts log into your Qredo account and click on the 'connect to MMI' button.",
+    );
+    expect(tokenContainer).toBeInTheDocument();
```
Comment (DDDDDanica) [inline ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.test.js:40]: Optional: you can find by testId.So if the text changes in the future, you don't need to modify this test ~

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js (hunk 2 above)
Comment (zone-live) [reply ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js]: Thank you!

File: ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.test.js (hunk 3 above)
Comment (zone-live) [reply ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.test.js:40]: Yap, but in this case we prefer to really check the text to make sure it's not changed in the future and has the correct custodian name, in this case Qredo.

**Comment by zone-live** [comment]
@metamaskbot update-policies

**Review by NidhiKJha — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**